data/Nanak360.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Nanak Bandyopadhyay",
+  "type": "personal",
+  "bio": "Full stack web developer 👨‍💻 | Web3 developer 🪙| JS & Python 💪",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "ReactJS",
+    "NodeJS",
+    "Angular"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/NBandyopadhyay" },
+    { "icon": "FaGithub", "url": "https://github.com/Nanak360" },
+    {
+      "icon": "FaFacebook",
+      "url": "https://www.facebook.com/nanak.bandyopadhyay"
+    }
+  ],
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/NBandyopadhyay",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/nanakbandyopadhyay/",
+      "icon": "FaLinkedinIn"
```

Comment:
```suggestion
      "icon": "FaLinkedin"
```

Reply:
![Screenshot 2023-04-17 at 10 51 28 AM](https://user-images.githubusercontent.com/36790357/232386708-8ece3d27-4ae9-46b5-aca8-63411f1977f7.png)
I'm using this icon